ferrocene/doc/safety-manual/src/degraded-environment.rst

```diff
@@ -24,35 +24,30 @@ undefined code generation. Therefore, the correctness of the Ferrocene
 toolchain installation needs to be verified before use. This is covered by
 constrain :id:`CSTR_0010_INSTALL`.
 
-The environment variables listed below were identified as potential sources of
-errors as they may lead to incorrect behavior of the Ferrocene toolchain. Users
-must verify that the values of these variables have not been set prior to using
-the Ferrocene toolchain.
-
-The following environment variables control or specify the inputs to the
+The following environment variables control or specify behavior of the
 compiler.
 
-.. list-table:: Input Environment Variables
+.. list-table:: Configuration Environment Variables
    :align: left
+   :header-rows: 1
 
-   * - ``HOST``
-   * - ``CARGO``
-   * - ``CARGO_ENCODED_RUSTFLAGS``
+   * - Environment Variable
+     - Description
 
-The following environment variables control or specify the inputs to the
-compiler.
+   * - ``RUST_MIN_STACK``
+   * - Sets the default stack size for spawned threads of the compiler.
+
+
+The environment variables listed below were identified as potential sources of
+errors as they may lead to incorrect behavior of the Ferrocene toolchain. Users
+must verify that the values of these variables have not been set prior to using
+the Ferrocene toolchain.
+
+The following environment variables may not be set prior to using the Ferrocene
+toolchian.
 
-.. list-table:: Output Environment Variables
+.. list-table:: Forbidden Environment Variables
```

Comment:
Should we replace this with a single list?

Reply:
You mean combining the two tables and then forbid the latter two vars in their description?

Reply:
Oh, I mean, replacing the second table with a bullet list (not sure why my brain wrote "single" there).